app/client/components/payment/update/stripeLogo.tsx

```diff
@@ -3,19 +3,16 @@ import React from "react";
 export const StripeLogo = () => (
   <svg width="119px" height="26px">
     <path
-      fill-rule="evenodd"
```

Comment:
this line was invalid html (copied from Stripe svg asset) - just noticed in this change so removing